samples/multicluster-razee/README.md

```diff
@@ -0,0 +1,481 @@
+# Notebook sample in multicluster
+
+This sample shows how Fybrik enables a Jupyter notebook workload to access a dataset across multicluster. It demonstrates how policies are seamlessly applied when accessing the dataset classified as financial data and how the dataset is transfered across multicluster based on policies.
+
+In this sample you play multiple roles:
+
+As a data owner you upload a dataset and register it in a data catalog
+As a data steward you setup data governance policies
+As a data user you specify your data usage requirements and use a notebook to consume the data.
+
+## Before you begin
+
+Ensure that you have the following installed on your machine:
+
+- git
+- make
+- jq
+- unzip
+- [Go](https://go.dev/) 1.16 or greater 
+- [Helm](https://helm.sh/) 3.7 or greater
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.18 or greater
+- [Kind](http://kind.sigs.k8s.io/)
+- A web browser
+
+## Setup the multicluster environment
+
+We leverage the scripts in the Fybrik repository to setup a [Razee](https://razee.io/) multicluster for this sample. First we download the repository:
+
+```bash
+git clone https://github.com/fybrik/fybrik.git
+cd fybrik
+```
+
+We want to have a use case that the dataset is transfered accross different clusters. So change the region of the data plane cluster `kind-kind` from `homeoffice` to `theshire`:
+
+```bash
+sed -i 's/homeoffice/theshire/g' ./charts/fybrik/kind-kind.values.yaml
+```
+
+Check whether the regions of the two clusters, `kind-control` and `kind-kind` are different:
+
+```bash
+grep 'region:' ./charts/fybrik/kind-control.values.yaml
+grep 'region:' ./charts/fybrik/kind-kind.values.yaml
+```
+
+Remove the use of the mockup images in control plane to trigger the built-in catalog and Open Policy Agent (OPA):
+
+```bash
+sed -i '/Connector:/d' ./charts/fybrik/kind-control.values.yaml
+sed -i '/-mock/d' ./charts/fybrik/kind-control.values.yaml
```

Comment:
does it make sense to have new values file for this multi-cluster run? (instead of changing the kind-control.values.yaml and kind-control.values.yaml)

Reply:
There are not many changes to `kind-control.values.yaml` and `kind-kind.values.yaml`:
4 lines for `kind-control.values.yaml`
1 line for `kind-kind.values.yaml`
If we have two new value files for multi-cluster, we will have to maintain them all the time in the future.
Even so, we still need update `hack/setup-local-multi-cluster.sh` to use the two new value files.
So in my opinion changing files is better than introducing new ones.

Reply:
I searched and not find any places that refer to `kind-control.values.yaml` and `kind-kind.values.yaml` except `setup-local-multi-cluster.sh` and `razee.sh`. It's seems these files are for multi-cluster cases dedicatedly. So maybe we can directly change  `kind-control.values.yaml`, `kind-kind.values.yaml` and `setup-local-multi-cluster.sh`?